diff --git a/node_modules/react-split-pane/index.d.ts b/node_modules/react-split-pane/index.d.ts
index d116f54..2bc0a3e 100644
--- a/node_modules/react-split-pane/index.d.ts
+++ b/node_modules/react-split-pane/index.d.ts
@@ -1,10 +1,12 @@
 import * as React from 'react';
+import { ReactNode } from "react"
 
 export type Size = string | number;
 
 export type Split = 'vertical' | 'horizontal';
 
 export type SplitPaneProps = {
+  children: ReactNode | ReactNode[];
   allowResize?: boolean;
   className?: string;
   primary?: 'first' | 'second';
@@ -23,6 +25,8 @@ export type SplitPaneProps = {
   paneStyle?: React.CSSProperties;
   pane1Style?: React.CSSProperties;
   pane2Style?: React.CSSProperties;
+  pane1ClassName?: string;
+  pane2ClassName?: string;
   resizerClassName?: string;
   step?: number;
 };
